bliss/data_augmentation.py

```diff
@@ -0,0 +1,117 @@
+import random
+
+import numpy as np
+import torchvision as TF
```

Comment:
Why TF?

Reply:
That is, `TF` seems like a potentially misleading name for `torchvision`.

How about instead doing `from torchvision.transforms import functional`?